commit c73b5ab54378ec6d7df9948c46ce87f98587af9a
Author: Tom Gundersen <teg@jklm.no>
Date:   2 days ago

    udev: remove redundant call to deprecated pci-db
    
    The binary is no longer shipped with systemd/udev. The functionality
    is taken over by a builtin, which is ran for all devices, rendering
    this rule superfluous.
    
    Cc: Jan de Groot <jgc@archlinux.org>
    Signed-off-by: Tom Gundersen <teg@jklm.no>

diff --git a/data/80-udisks.rules b/data/80-udisks.rules
index d1c8c5d..4bf2b37 100644
--- data/80-udisks.rules
+++ data/80-udisks.rules
@@ -1,10 +1,6 @@
 # Do not edit this file, it will be overwritten on updates
 #
 
-# import names for PCI storage controllers
-#
-SUBSYSTEM=="pci", ACTION=="add|change", ENV{ID_MODEL_FROM_DATABASE}=="", ATTR{class}=="0x01*", IMPORT{program}="pci-db %p"
-
 # Set eSATA port type for known eSATA CardBus adapters - first we want to ensure
 # the device is on a cardbus controller (upper PCI device) - then we check
 # vid/pid (lower PCI device)
